docs/general/ddl/fulltext-indices.rst

```diff
@@ -216,7 +216,7 @@ Here is a simple Example::
 .. hide: Test table creation with custom analyzer::
 
     cr> create table hidden_test_table (
-    ...     fc text index using fulltext with(analyzer=myanalyzer)
+    ...     fc text index using fulltext with (analyzer = 'myanalyzer')
```

Comment:
This worked by accident. Maybe requires a (breaking?) changes entry?

Reply:
Not sure, it looks more like a bugfix to me than a breaking change.

Reply:
yeah but it also worked before. The scope of the change is also too big for a hotfix. I'd even hold it off for 5.4 and merge it after it's released. 

I currently tend towards a regular changes entry, mentioning that it became stricter?

Reply:
👍 Fine for me.

Reply:
:+1: for me